vcf2maf.pl

```diff
@@ -233,10 +228,38 @@ sub GetBiotypePriority {
     'retain-ann=s' => \$retain_ann,
     'min-hom-vaf=s' => \$min_hom_vaf,
     'remap-chain=s' => \$remap_chain,
-    'max-subpop-af=f' => \$max_subpop_af
+    'max-subpop-af=f' => \$max_subpop_af,
+    'samtools-exec=s' => \$samtools,
+    'tabix-exec=s' => \$tabix,
+    'liftover-exec=s' => \$liftover,
+
 ) or pod2usage( -verbose => 1, -input => \*DATA, -exitval => 2 );
+
+if( $man ) {
```

Comment:
the `--man` detection is moved up as it should have higher precedence than `--help`

Also,